app/jobs/parse_sorn_xml_job.rb

```diff
@@ -5,6 +5,7 @@ def perform(sorn_id)
     sleep 1
 
     sorn = Sorn.find(sorn_id)
+    sorn.get_action_type
```

Comment:
I moved the call to update the action type to ParseSornXmlJob. This job is starting to become where the SORN updates all of its data, not just parse the xml. I'm adding the references to other SORNs check [here in another PR](https://github.com/18F/all_sorns/pull/43/files#diff-2aa53c4843181cd73c9199b3a41a0cf095c6d57d97554ff858f8ebdbb9776fd8R11).

We should probably rename the job since its starting to do more work.